.github/workflows/ci_cmake.yml

```diff
@@ -0,0 +1,135 @@
+      - '*.md'
+      - '*.yml'
+      - 'wxwidgets.props'
+
+jobs:
+  build:
+    runs-on: ${{ matrix.runner }}
+    name: ${{ matrix.name }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: Ubuntu 18.04 wxGTK 3
+            runner: ubuntu-18.04
+            cmake_generator: Unix Makefiles
+          - name: macOS 10.15 wxOSX
+            runner: macos-10.15
+            cmake_generator: Xcode
+            cmake_defines: -DCMAKE_CXX_STANDARD=11
+          - name: macOS 10.15 wxIOS
+            runner: macos-10.15
+            cmake_generator: Xcode
+            cmake_defines: -DCMAKE_SYSTEM_NAME=iOS -DCMAKE_FIND_ROOT_PATH=/usr/local -DCMAKE_XCODE_ATTRIBUTE_CODE_SIGNING_ALLOWED=NO
+            cmake_samples: OFF
+            cmake_tests: OFF
+
+    env:
+      wxGTK_VERSION: ${{ matrix.gtk_version && matrix.gtk_version || 3 }}
+      wxCMAKE_GENERATOR: ${{ matrix.cmake_generator }}
+      wxCMAKE_DEFINES: ${{ matrix.cmake_defines }}
+      wxCMAKE_TESTS: ${{ matrix.cmake_tests }}
+      wxCMAKE_SAMPLES: ${{ matrix.cmake_samples }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: 'recursive'
+
+      - name: Set environment variables
+        run: |
+          . build/tools/proc_count.sh
+          wxPROC_COUNT=`proc_count`
+          if [ "$wxCMAKE_GENERATOR" == "Xcode" ]; then
+              wxBUILD_ARGS="-jobs $wxPROC_COUNT -quiet"
+          else
+              wxBUILD_ARGS=-j$wxPROC_COUNT
+          fi
+          echo wxBUILD_ARGS=$wxBUILD_ARGS >> $GITHUB_ENV
+
+          if [ -z $wxCMAKE_TESTS ]; then wxCMAKE_TESTS=CONSOLE_ONLY; fi
+          echo wxCMAKE_TESTS=$wxCMAKE_TESTS >> $GITHUB_ENV
+          if [ -z $wxCMAKE_SAMPLES ]; then wxCMAKE_SAMPLES=SOME; fi
+          echo wxCMAKE_SAMPLES=$wxCMAKE_SAMPLES >> $GITHUB_ENV
+
+          # Setting this variable suppresses "Error retrieving accessibility bus address"
+          # messages from WebKit tests that we're not interested in.
+          echo NO_AT_BRIDGE=1 >> $GITHUB_ENV
+
+      - name: Before install
+        run: |
+          ./build/tools/before_install.sh
+
+      - name: Configuring
+        run: |
+          cmake --version
+          mkdir build_cmake
+          pushd build_cmake
+          cmake -G "$wxCMAKE_GENERATOR" $wxCMAKE_DEFINES -D wxBUILD_SAMPLES=$wxCMAKE_SAMPLES -D wxBUILD_TESTS=$wxCMAKE_TESTS ..
+
+      - name: Building
+        working-directory: build_cmake
+        run: |
+          cmake --build . -- $wxBUILD_ARGS
+
+      - name: Installing
+        working-directory: build_cmake
+        run: |
+          sudo cmake --build . --target install -- $wxBUILD_ARGS
+
+      - name: Testing
+        if: matrix.cmake_tests != 'OFF'
+        working-directory: build_cmake
+        run: |
+          . ../build/tools/httpbin.sh
+
+          httpbin_launch
+
+          ctest -V -C Debug -E "test_drawing" --output-on-failure --interactive-debug-mode 0 . || rc=$?
```

Comment:
Could you please add a comment explaining why is `test_drawing` excluded? I'd like to know if it's worth trying to run it in the other CI builds...

Reply:
I added it to keep the AppVeyor and Travis scripts the same. I can remove it. Only `base_test` is built, so it doesn't do anything here.

Reply:
I didn't notice AppVeyor did the same thing (Travis one doesn't exist any longer), so it's as you wish, really, but if it doesn't change anything I think it would make more sense to remove it from the other script rather than add here.

Reply:
I removed it here. AppVeyor builds all tests by default, and then `ctest` will run all available tests. I don't remember the reason why `test_drawing` is excluded from running.